casper/src/main/resources/PoS.rhox

```diff
@@ -531,7 +544,86 @@ new  PoS,
                   }
                }
              }
+            } |
+
+            // Initialize pending rewards state for each validator
+            contract PendingRewards(@"init", @validatorKeys, @return) = {
```

Comment:
The argument `validatorKeys` is unused.  Maybe you meant to use it instead of `initialActive` below?

Reply:
It seems that tests are to simple. This is a BUG.